spring-cloud-gcp-data-spanner/src/main/java/org/springframework/cloud/gcp/data/spanner/core/admin/SpannerDatabaseAdminTemplate.java

```diff
@@ -93,14 +93,18 @@ public void executeDdlStrings(Iterable<String> ddlStrings,
 			if (createDatabase) {
 				this.databaseAdminClient
 						.createDatabase(getInstanceId(), getDatabase(), ddlStrings)
-						.waitFor();
+						.get();
 			}
 			else {
 				this.databaseAdminClient.updateDatabaseDdl(getInstanceId(), getDatabase(),
-						ddlStrings, null).waitFor();
+						ddlStrings, null).get();
 			}
 		}
-		catch (SpannerException e) {
+		catch (InterruptedException e) {
```

Comment:
Not related to this change, just for the future consideration: there is a rule in the upstream Checkstyle config which disallows a single letter identifiers. So, would be better to keep that in mind and start follow such a practice and don't bump to the pain to fix all of them at once, when we come to the mentioned config.

Thanks for understanding.